Packs/ServiceNow/README.md

```diff
@@ -1,16 +1,42 @@
-Cortex XSOAR interfaces with ServiceNow to help streamline security-related service management and IT operations. 
-
-The data in ServiceNow tickets can be mirrored to Cortex XSOAR so that you can track the status and information in the task. 
-You can also provide comments or attachments in XSOAR which will appear in ServiceNow. 
-
-
-# What does this pack do?
-
-- View, create, update, or delete a ServiceNow ticket directly from the Cortex XSOAR and enrich it with Cortex XSOAR data.
-- View, create, update, and delete records from any ServiceNow table.
-- Query ServiceNow data with the ServiceNow query syntax.
-
-As part of this pack, you will also get 2 out-of-the-box layouts so that you can visualize ServiceNow ticket information in Cortex XSOAR.
-
-
-The Create ServiceNow Ticket playbook provides an example for how to use the Mirror ServiceNow Ticket playbook to mirror data and ServiceNow Ticket State Polling sub-playbook to track when the ticket closes.
+Cortex XSOAR interfaces with ServiceNow to help streamline security-related service management and IT operations. 
+
+The data in ServiceNow tickets can be mirrored to Cortex XSOAR so that you can track the status and information in the task. 
+You can also provide comments or attachments in XSOAR which will appear in ServiceNow. 
+
+
+# What does this pack do?
+
+- View, create, update, or delete a ServiceNow ticket directly from the Cortex XSOAR and enrich it with Cortex XSOAR data.
+- View, create, update, and delete records from any ServiceNow table.
+- Query ServiceNow data with the ServiceNow query syntax.
+
+As part of this pack, you will also get 2 out-of-the-box layouts so that you can visualize ServiceNow ticket information in Cortex XSOAR.
```

Comment:
```suggestion
As part of this pack, you will also get two out-of-the-box layouts so that you can visualize ServiceNow ticket information in Cortex XSOAR.
```